bin/mozilla/aa.pl

```diff
@@ -1880,7 +1895,7 @@ sub search {
 	      </tr>
 	      $vc
 	      $invnumber
-	      <tr>
```

Comment:
what about the removed tr? Should it be added after $currency?

Reply:
I have incorrectly used <tr> when it should be </tr> when defining $currency on line 1811. Please change closing <tr> to </tr>

You have sharp eye.